platform/linux-generic/m4/odp_crypto.m4

```diff
@@ -0,0 +1,38 @@
+# ODP_CRYPTO
+# ----------
+# Select default crypto implementation
+AC_ARG_WITH([crypto],
+            [AS_HELP_STRING([--with-crypto],
+                            [Choose crypto implementation (openssl/armv8crypto/null)]
+                            [[default=with] (linux-generic)])],
+            [], [with_crypto=openssl])
+
+# Default to OpenSSL implementation if crypto is enabled
+AS_IF([test "x$with_crypto" = "xyes"], [with_crypto=openssl])
+
+# Default to Null implementation if crypto is disabled
+AS_IF([test "x$with_crypto" = "xno"], [with_crypto=null])
+AS_IF([test "${with_openssl}" = "no"], [with_crypto=null])
+
+AS_IF([test "x$with_crypto" != "xopenssl" -a "x$with_crypto" != "xarmv8crypto" -a "x$with_crypto" != "xnull"],
+      [AC_MSG_ERROR([Invalid crypto implementation name])])
+
+##########################################################################
+# OpenSSL implementation
+##########################################################################
+AM_CONDITIONAL([WITH_OPENSSL], [test "x$with_crypto" == "xopenssl"])
+
+##########################################################################
+# ARMv8 Crypto library implementation
+##########################################################################
+AS_IF([test "x$with_crypto" == "xarmv8crypto"],
+      [PKG_CHECK_MODULES([AARCH64CRYPTO], [libAArch64crypto])
+       AARCH64CRYPTO_PKG=", libAArch64crypto"
+       AC_SUBST([AARCH64CRYPTO_PKG])])
+AM_CONDITIONAL([WITH_ARMV8_CRYPTO], [test "x$with_crypto" == "xarmv8crypto"])
+
+##########################################################################
+# Null implementation
+##########################################################################
+AS_IF([test "x$with_crypto" == "xnull"],
+      [AC_MSG_WARN([Using null crypto. Strong cryptography is not available])])
```

Comment:
Please remove the following warning from `platform/linux-generic/m4/odp_openssl.m4`:
```
AS_IF([test "${with_openssl}" = "no"],
      [AC_MSG_WARN([Strong cryptography is not available without OpenSSL])])
```

Reply:
Okay, I will remove this check and warning. Thanks!

Reply:
The latest version did the opposite of what I requested.

This was supposed to stay:
```
# Null implementation
##########################################################################
AS_IF([test "x$with_crypto" == "xnull"],
      [AC_MSG_WARN([Using null crypto. Strong cryptography is not available])])
```
And this can be removed (platform/linux-generic/m4/odp_openssl.m4):
```
AS_IF([test "${with_openssl}" = "no"],
      [AC_MSG_WARN([Strong cryptography is not available without OpenSSL])])
```

This way the warning is only printed if we don't have openssl or armv8 crypto.